src/xdocs/checks/javadoc/javadocparagraph.xml.template

```diff
@@ -29,6 +29,14 @@
             nested paragraph tags are allowed to do that.
           </li>
         </ul>
+        <p>
+          <b>ATTENTION:</b>
+          <ul>
```

Comment:
Hmm, this is not allowed tag ... Strange.
Can we workaround it somehow?
https://ci.appveyor.com/project/Checkstyle/checkstyle/builds/50771010/job/410wglmmfwmaa7nr#L254

Reply:
You can reproduce on local by `./.ci/validation.cmd site_without_verify`

Or `mvn site`